### ISSUE TITLE
RS expansion not returns address prefix ranges?

### ISSUE THREAD

**Issue by sumkincpp** [irrdnet/irrd#543]
**Describe the bug**
It seems, that RS-Expansion does not return address prefix ranges. 

**To Reproduce**
```
$ echo '!v' | nc rr.ntt.net 43
A22
IRRd -- version 4.1.4
C
$ echo '!iRS-36327' | nc rr.ntt.net 43
A409
142.4.128.0/19^19-24 192.211.128.0/17^17-24 198.100.192.0/18^18-24 198.41.80.0/20^20-24 199.167.80.0/21^21-24 199.27.88.0/22^22-24 199.48.72.0/21^21-24 199.66.160.0/21^21-24 208.68.0.0/22^22-24 208.69.64.0/21^21-24 208.73.128.0/21^21-24 208.80.240.0/21^21-24 208.86.100.0/22^22-24 208.88.216.0/21^21-24 208.95.24.0/21^21-24 216.80.5.0/24 216.80.55.0/24 216.80.6.0/24 23.232.0.0/17^17-24 74.119.152.0/21^21-24
C
$ echo '!iRS-36327,1' | nc rr.ntt.net 43
A43
216.80.5.0/24 216.80.55.0/24 216.80.6.0/24
C
```

**Expected behaviour**

This expansion should also return address prefix ranges
```
$ echo '!iRS-36327,1' | nc rr.ntt.net 43
```

**RADB**: works as expected

**IRRd version you are running**
4.1.4

**Relevant places**
Members are being tested as IP-Addresses:
https://github.com/irrdnet/irrd/blob/c904fb94f5854f05069a7ca337494c464adee8ce/irrd/server/query_resolver.py#L214

**Comment by mxsasha** [comment]
This indeed looks like a bug and you are right about where it is. Curious, because this was introduced in 86baffd3dec0b715ee6c49675bf9e2af7f634b90 which was already in 4.0, and we did run comparisons between legacy IRRd and this. Must be somewhat rare.

**Comment by mxsasha** [comment]
Still needs backport to 4.1

**Comment by mxsasha** [comment]
Released in 4.1.6, thanks!
